src/components/misc/personViews/PersonViewTable.jsx

```diff
@@ -132,7 +132,24 @@ export default class PersonViewTable extends React.Component {
                             const val0 = row0.data.content[this.state.sortIndex] || '';
                             const val1 = row1.data.content[this.state.sortIndex] || '';
 
-                            let x = val0.toString().localeCompare(val1.toString());
+                            let x = 0;
+
+                            if (Array.isArray(val0) && typeof(val0[0]) != 'undefined' && typeof(val1[0]) != 'undefined') {
```

Comment:
I think it would be easier to read if you separate the array check from the `undefined` check (also see comments on lines 144-152.